openstf/pipeline/create_forecast_sklearn.py

```diff
@@ -1,52 +1,55 @@
 # SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
 #
 # SPDX-License-Identifier: MPL-2.0
+from typing import Union
 from datetime import datetime, timedelta, timezone
+from pathlib import Path
 
 import pandas as pd
 import structlog
-
-# from ktpbase.config.config import ConfigManager
+from sklearn.base import RegressorMixin
 
 from openstf.validation import validation
 from openstf.feature_engineering.feature_applicator import (
     OperationalPredictFeatureApplicator,
 )
 from openstf.model.confidence_interval_applicator import ConfidenceIntervalApplicator
-from openstf.preprocessing import preprocessing
+
 from openstf.model.serializer import PersistentStorageSerializer
 from openstf.postprocessing.postprocessing import (
     add_prediction_job_properties_to_forecast,
 )
 from openstf.model.fallback import generate_fallback
 
 
-# TODO add loading of model to task
-# # Load most recent model for the given pid
-# model = PersistentStorageSerializer(
-#     trained_models_folder=MODEL_LOCATION
-# ).load_model(pid=pj["id"])
+def create_forecast_pipeline(
+    pj: dict, input_data: pd.DataFrame, trained_models_folder: Union[str, Path]
+) -> pd.DataFrame:
+    """Midle level pipeline that takes care of all persistent storage dependencies
 
-# def create_forecast_pipeline(pj, input_data, trained_models_folder=None):
-#     logger = structlog.get_logger(__name__)
-#     config = ConfigManager.get_instance()
+    Args:
+        pj (dict): Prediction job
+        input_data (pd.DataFrame): Raw training input data
+        trained_models_folder (Path): Path where trained models are stored
 
-#     # Use default if not given. ConfigManager ??
-#     if trained_models_folder is None:
-#         trained_models_folder = config.paths.trained_models_folder
 
-#     # Load most recent model for the given pid
-#     model = PersistentStorageSerializer(
-#         trained_models_folder=trained_models_folder
-#     ).load_model(pid=pj["id"])
+    Returns:
+        pd.DataFrame with the forecast
 
-#     forecast = create_forecast_pipeline_core(pj, input_data, model)
+    """
+    # Load most recent model for the given pid
+    model = PersistentStorageSerializer(
+        trained_models_folder=trained_models_folder
+    ).load_model(pid=pj["id"])
 
-#     # TODO write forecast to db ???
+    return create_forecast_pipeline_core(pj, input_data, model)
 
 
-def create_forecast_pipeline_core(pj, input_data, model):
+def create_forecast_pipeline_core(
+    pj: dict, input_data: pd.DataFrame, model: RegressorMixin
+) -> pd.DataFrame:
     """Computes the forecasts and confidence intervals given a prediction job and input data.
```

Comment:
Same the `summary` line and the `description` are mixed here: 
![image](https://user-images.githubusercontent.com/6715707/121557714-44a94500-ca15-11eb-896e-26aa193f22a8.png)